aten/src/ATen/core/ATenDispatch.cpp

```diff
@@ -7,37 +7,31 @@ ATenDispatch & globalATenDispatch() {
   return singleton;
 }
 
-void* ATenOpTable::getFallbackOp(TensorTypeId tid) const {
-  // TODO: an alternate strategy here would be to mask out the dead key
-  // and then redispatch gain (automatic delegation).  I haven't done this
-  // for now to make it easier to smoke out error cases.
-  if (function_table_[static_cast<int64_t>(TensorTypeId::UndefinedTensorId)] == nullptr) {
-    std::ostringstream oss;
-    bool first = true;
-    for (int64_t i = 0; i < static_cast<int64_t>(TensorTypeId::NumTensorIds); i++) {
-      if (function_table_[i] != nullptr) {
-        if (!first) oss << ", ";
-        oss << toString(static_cast<TensorTypeId>(i));
-        first = false;
-      }
+void ATenOpTable::reportError(TensorTypeId tid) const {
```

Comment:
this doesn't seem to report anything? It just discards the constructed string?

Reply:
It raises an exception.

Reply:
oh I somehow thought line 19 was ending this function already, missed that it continues